tests/test_cli.py

```diff
@@ -33,6 +33,23 @@ def test_cli_createuser(app):
     result = runner.invoke(users_create, ["not-an-email", "--password", "123456"])
     assert result.exit_code == 2
 
+    # Create use with profile that is not valid JSON
+    result = runner.invoke(
+        users_create, ["test@example.com", "--profile", "<json>profile</json>"]
+    )
+    assert result.exit_code != 0
+
+    # Create use with profile that will not validate
+    result = runner.invoke(
+        users_create,
+        [
+            "test@example.com",
+            "--profile",
+            '{"full_name" : "test_user", "extra_data" : "not_allowed"}',
+        ],
+    )
+    assert result.exit_code != 0
```

Comment:
minor: I think we also need a test-case for the happy path, i.e. passing a correct `--profile` value that then gets set on the created user.

Reply:
coordination: @slint please take a look if  the test pass please approve?